src/transformers/onnx/features.py

```diff
@@ -155,6 +156,16 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=BertOnnxConfig,
         ),
+        "bigbird": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            # "multiple-choice",
```

Comment:
I realise we have this feature commented out for other models, but I suggest removing this for now. I've opened an issue here to track this: https://github.com/huggingface/transformers/issues/16695

Reply:
Thanks!